client/packages/host/webpack.config.js

```diff
@@ -143,7 +143,7 @@ module.exports = env => {
     plugins: [
       new ReactRefreshWebpackPlugin(),
       new webpack.DefinePlugin({
-        FEATURE_PACK_VARIANTS: env.FEATURE_PACK_VARIANTS,
+        FEATURE_EXAMPLE: env.FEATURE_EXAMPLE,
```

Comment:
Left this in here, but would be nice to have a more fully baked example? Maybe we need to add an issue for that?